PathKit/PathKit_tvOS/PathKit_tvOS.h

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2019 JABT Labs. All rights reserved.
+//
+// This file is part of PathKit. The full PathKit copyright notice,
+// including terms governing use, modification, and redistribution, is
+// contained in the file LICENSE at the root of the source code distribution
+// tree.
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for PathKit_tvOS.
+FOUNDATION_EXPORT double PathKit_tvOSVersionNumber;
```

Comment:
We can resume the same header for every platform.

Reply:
Did you mean reuse the same header? If so I'm not sure how to do that. Can you show me how? Or send me a resource that shows how to do that?

Reply:
There is nothing to it. Just delete duplicates and add to all targets as a public header.

Reply:
Ah my bad, I was over complicating things. Thought there was more to it.